src/Pages/Video/video-detail.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import "./video-detail.css";
+import { useVideo } from "../../Context/video-context";
+import { BsHeart } from "react-icons/bs";
+
+export function VideoDetail() {
+  const { video } = useVideo();
+  const { videoId } = useParams();
+  const matchId = (item) => item._id === videoId;
+  let item = video.find(matchId);
+
+  return (
+    <div
+      style={{
+        backgroundImage: `url(${item.banner})`,
+      }}
+      className="bg-responsive"
+    >
+      <div className="flex flex-column bg-div bg-blur">
+        {" "}
+        <div className="flex vid-desc-flex align-item-center">
+          <div className="color-white w-50 vid-desc">
+            <h2 className="color-white vid-title">{item.title}</h2>
+            {item.season && item.episode ? (
+              <h5 className="vid-info">
+                {" "}
+                {item.season} Season, {item.episode} Episodes
+              </h5>
+            ) : (
+              ""
+            )}
```

Comment:
suggestion: here you can do `&&` instead of `?:` since you are not returning anything in `else` case